dataland-email-service/src/main/resources/email-templates/data_access_requested.html.ftl

```diff
@@ -146,7 +146,7 @@ message
     <tr>
         <td style="text-align: left; padding:0; margin:0; border: 0; height: 54px; width: 26px"></td>
         <td style="background-color: #ff5c00; text-align: center; padding:0; margin:0; border: 0; height: 54px; width: 468px;">
-            <a href="${baseUrl}/companies/${companyId}" target="_blank" style="border: 0 none; line-height: 30px; color: #ffffff; font-size: 18px; width: 100%; display: block; text-decoration: none;">
+            <a href="${baseUrl}/companyrequests" target="_blank" style="border: 0 none; line-height: 30px; color: #ffffff; font-size: 18px; width: 100%; display: block; text-decoration: none;">
```

Comment:
Die E-Mail die aktuell versand wird mit "You have now access" hat einen Button "Review the provided data" welcher auf die Seite des data requests führt. Sollte das nicht eher auf die Seite des Datensatzes führen?

Reply:
@LaurinStrelow kümmert sich um diesen Punkt

Reply:
Ist erledigt.